src/pages/settings/change-email/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useSupabaseClient } from "@supabase/auth-helpers-react"
+import { Database } from "@/types/database"
+import React, { useEffect, useState } from "react"
+import { FormProvider, useForm } from "react-hook-form"
+import { ArrowLeftIcon } from "@heroicons/react/24/outline"
+import { useRouter } from "next/router"
+import { toast } from "react-toastify"
+import ChangeEmailForm from "@/components/settings/account/change-email/change-email-form"
+import ChangeEmailConfirmation from "@/components/settings/account/change-email/change-email-confirmation"
+
+export interface ChangeEmailInputs {
+  newEmailAddress: string
+  currentPassword: string
+}
+
+const Index = () => {
```

Comment:
Function name could reflect the name of the page. Though could be ignored as pages are rarely imported in other files. 

Occurs in different page files as well.
![image](https://github.com/ASVGay/the-rhapsodies/assets/4838158/56b57f10-9c7b-450f-8e10-87e24cc5d504)

Reply:
If this is desired, it would be better to create a separate PR for it to refactor all page names imo